packages/perseus/src/perseus-types.ts

```diff
@@ -14,6 +14,88 @@ type StyleParams = {
 // TODO(FEI-5054): Figure out how to get global .d.ts files working with monorepos
 type Empty = Record<never, never>;
 
+export type WidgetsType = {
```

Comment:
I am fully open to hearing any alternative naming suggestions here. I feel like our Perseus type names are incredibly confusing currently.

Reply:
I ended up going with "PerseusWidgets" but I'm still not the happiest with the naming clarity.